sw/airborne/modules/ctrl/approach_moving_target.c

```diff
@@ -54,13 +75,15 @@ Butterworth2LowPass target_heading_filt;
 #include <stdio.h>
 
 struct Amt amt = {
-  .distance = 60,
-  .speed = -1.0,
-  .pos_gain = 0.3,
+  .distance = APPROACH_MOVING_TARGET_DISTANCE,
+  .speed = APPROACH_MOVING_TARGET_SPEED,
   .psi_ref = 180.0,
-  .slope_ref = 35.0,
-  .speed_gain = 1.0,
-  .relvel_gain = 1.0,
+  .slope_ref = APPROACH_MOVING_TARGET_SLOPE,
+  .err_slowdown_gain = APPROACH_MOVING_TARGET_ERR_SLOWDOWN_GAIN,
+  .pos_gain = APPROACH_MOVING_TARGET_POS_GAIN,
+  .speed_gain = APPROACH_MOVING_TARGET_SPEED_GAIN,
+  .relvel_gain = APPROACH_MOVING_TARGET_RELVEL_GAIN,
+  .cutoff_freq_filters_hz = CUTOFF_FREQ_FILTERS_HZ,
```

Comment:
Doesn't have the prefix `APPROACH_MOVING_TARGET` and also no documentation etc.

Reply:
done